core/sched.c

```diff
@@ -101,7 +101,7 @@ int __attribute__((used)) sched_run(void)
     }
 
 #ifdef MODULE_SCHEDSTATISTICS
-    unsigned long time = _xtimer_now();
+    unsigned long long time = _xtimer_now64();
```

Comment:
a better name for this variable would be `now` or something. `time()` is a POSIX standard function in time.h so it's better to avoid that name for clarity.